Docs/seo updates (#36083)
hashicorp/terraform: website/docs/intro/core-workflow.mdx

---
page_title: Overview of the core Terraform workflow

> description: Learn how to provision and manage infrastructure using the core Terraform workflow for individuals, teams, and organizations.

Reviewer: ```suggestion
description: Learn how to individuals, teams, and organizations can use the Terraform workflow to provision and manage infrastructure.
```
